langs/assembly/Dockerfile

```diff
@@ -1,10 +1,10 @@
 FROM node:22-alpine3.20 AS builder
 
-ENV VERSION=2.4.2
+ENV VERSION=2.4.3
 
 RUN apk add --no-cache binutils
 
-RUN npm install -g @defasm/cli@1.2.3 @defasm/core@$VERSION
+RUN npm install -g @defasm/cli@1.3.0 @defasm/core@$VERSION
```

Comment:
This didn't work when I tried it, did you try it?

Reply:
I did not, and I should've known better. Please hold.

Reply:
Interesting stuff. I'm assuming you're the one in charge of keeping `npm` up-to-date? Making 127.0.0.1 produces a lovely notice, but I'm not convinced it will fix things.

```bash
assets-1  | npm notice New patch version of npm available! 10.8.2 -> 10.8.3
assets-1  | npm notice Changelog: https://github.com/npm/cli/releases/tag/v10.8.3
assets-1  | npm notice To update run: npm install -g npm@10.8.3
```

No issues when reverting to `1.2.3`, but `1.2.4` is also failing. Awesome stuff. I'll revert to `1.2.3` for now, and take a deeper look when I'm not building/rebuilding four different Dockerfiles simultaneously.